src/Tracking/Conversions.php

```diff
@@ -266,6 +266,9 @@ private function get_search_data( Search $data ) {
 	 */
 	private function send_request( string $event_name, array $data ) {
 		$ad_account_id = Pinterest_For_WooCommerce()::get_setting( 'tracking_advertiser' );
+		if ( empty( $ad_account_id ) ) {
```

Comment:
Should this maybe happen at the `Tracking:track_event` level? The tag should also not be sent if there is no ad_account_id.

Reply:
I thought about stopping any tracking in case account_id is missing. I thought that even if Tag does not require account_id it would still make sense to stop the events because there is something wrong.

But it looks like this is good so we can roll with that.